integreat_cms/cms/templates/languages/language_form.html

```diff
@@ -54,7 +54,13 @@ <h3 class="font-bold text-black">
                         </div>
                         <label for="{{ form.bcp47_tag.id_for_label }}" class="block mb-2">{{ form.bcp47_tag.label }}</label>
                         {% render_field form.bcp47_tag|add_error_class:"border-red-500" %}
-                        <div class="mb-2 text-s text-gray-600">{{ form.bcp47_tag.help_text }}</div>
+                        <div class="mb-2 text-s text-gray-600">
+                            {{ form.bcp47_tag.help_text }}
+                            </ br>
+                            {% if DEEPL_ENABLED %}
+                                <i class="mr-2" icon-name="alert-triangle"></i>{% translate "For some language DeepL needs a correct BCP47 code (for example English or Portuguese). For exact information which languages require this, please checkout the" %} <a href="https://www.deepl.com/de/docs-api/translate-text/translate-text/">{% translate "DeepL documentation" %}</a>.
```

Comment:
```suggestion
                            {% if DEEPL_ENABLED %}
                                <br />
                                <br />
                                <i class="h-5" icon-name="alert-triangle"></i>
                                {# djlint:off #}
                                {% blocktranslate trimmed with link="href='https://www.deepl.com/de/docs-api/translate-text/translate-text/' target='_blank' class='underline'" %}
                                    For some languages, DeepL needs a correct BCP47 code (for example English or Portuguese).
                                    For more information, please checkout the <a {{ link }}>DeepL documentation</a>.
                                {% endblocktranslate %}
                                {# djlint:on #}
```
Two problems with this part:
1. `</ br>` is an incorrect tag
2. Please do not split translated text into chunks - always keep the text as a whole to make it possible to translate the sentence in context, and also the position of a specific part of the sentence may be different in other languages.